README.md

```diff
@@ -8,7 +8,7 @@ The future face of osu!.
 Requirements
 ------------
 
-- A PHP 7.1+ (or equivalent) environment
+- A PHP 7.2 (or equivalent) environment
```

Comment:
What I'm saying is it should be `7.1` instead of `7.1+` because on of the libraries is broken in 7.2 ಠ_ಠ